test/extended/builds/pipeline.go

```diff
@@ -200,7 +201,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			br.AssertSuccess()
 
 			g.By("confirm all the log annotations are there")
-			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
+			err = jenkins.ProcessLogURLAnnotations(oc, br, !useSnapshotImage)
```

Comment:
it won't verify the blue ocean annotation, but will the other 2

certainly long term there's the argument to move all the PR testing to v2; but in the past I've deferred because:
a) until the recent v1 deprecation decision, it seemed good to have some level of v1 testing being done
b) in the past when I tried switching to PR test image to extend v2, the docker build blew up (precise errors escape me but as I recall it wasn't a simple sort of fix)

My preference would be to defer on blueocean annotation verification for the PR testing for now, and then when we actually start making the changes to officially deprecate v1, circle back to switching all 3 plugin PR testers to v2, and then sort through any issues that might exist there.

Thoughts?